__tests__/src/hooks/useAllTickets.test.ts

```diff
@@ -1,17 +1,15 @@
 import useAllTickets from '@hooks/useAllTickets';
-import * as DataService from '@services/DataService';
 import { renderHook, waitFor } from '@testing-library/react';
 import { mockServiceRequestSummaries } from '@utils/TestData';
 
 jest.mock('@services/DataService');
-const mockedDataService = jest.mocked(DataService);
 
 it('returns the tickets from the db', async () => {
   //* Act
   const { result } = renderHook(useAllTickets);
   //* Assert
   await waitFor(() => {
-    expect(mockedDataService.getServiceRequestSummary).toHaveBeenCalled();
-    expect(result.current).toEqual(mockServiceRequestSummaries);
+    expect(result.current.isLoading).toBe(false);
   });
+  expect(result.current.data).toEqual(mockServiceRequestSummaries);
```

Comment:
Re-read react-testing-lib documentation and awaitFor is just there to pause while async stuff happens, it shouldn't actually contain the tested assertions (though it could)

https://testing-library.com/docs/dom-testing-library/api-async/#waitfor